MapboxAndroidDemo/src/main/res/values/descriptions_strings.xml

```diff
@@ -72,6 +72,7 @@
     <string name="activity_camera_bounding_box_description">Position the camera so that all the given markers are in view.</string>
     <string name="activity_camera_restrict_description">Prevent a map from being panned to a different place.</string>
     <string name="activity_camera_slowly_rotating_description">Slowly have the camera circle around a single point.</string>
+    <string name="activity_camera_zoom_to_cluster_children_description">Tap on a cluster and adjust the map camera bounds to fit that cluster\'s data.</string>
```

Comment:
I'm not exactly sure what the wording here _should_ be, but I think the description would benefit from having more specificity than just the term "data." I'm not sure that it's intuitive what it means to say "adjust the map camera bounds to fit that cluster's data" unless it's also clarified that the cluster is expanding into its leaves.

Reply:
Fixed to `Tap on a cluster and adjust the camera bounds to show the cluster's leaves.`